cmake/package_general.cmake

```
@@ -72,7 +72,7 @@ install(RUNTIME_DEPENDENCY_SET OPENMS_DEPS
         DESTINATION ${INSTALL_LIB_DIR}
         COMPONENT Dependencies
         PRE_EXCLUDE_REGEXES "api-ms" "ext-ms" "hvsi" "pdmutilities"
-        DIRECTORIES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+        DIRECTORIES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} $<TARGET_FILE_DIR:OpenMS>)
```

```suggestion
        DIRECTORIES $<TARGET_FILE_DIR:OpenMS>)
```
I would probably remove the other path. Since they may evaluate to the same path. Not sure if CMake resolves duplicates and if it does not you get at least a warning, if not an error since the libs are found multiple times.